### PR TITLE
Improve stubs in av.container

### PR DESCRIPTION
**Pull request by laggykiller**

### CODE REVIEW

File: av/container/output.pyi [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
 from .core import Container
 
 class OutputContainer(Container):
     def __enter__(self) -> OutputContainer: ...
+    def add_stream(
+        self,
+        codec_name: str,
+        rate: Fraction | int | float,
+        template: Stream,
```
Comment (WyattBlue) [inline av/container/output.pyi:15]: Should be `template: Stream | None = None`

Comment (laggykiller) [reply av/container/output.pyi:15]: I see that either codec_name or template is needed. Should I type it as an overload function instead?

Comment (WyattBlue) [reply av/container/output.pyi:15]: Yes, overload is appropriate here

Comment (laggykiller) [reply av/container/output.pyi]: Sorry I edited the previous comment, I found out that typing as overloading might be impossible

**Pull request merged**